### PR TITLE
Vet clinic DB: query multiple tables

### PR DESCRIPTION
**Pull request by gtekle**
## What this PR does:
 - Created a table named owners with (id, full_name, age) columns. 
 - Created a table named species with (id, name) columns. 
 - Modified animals table as follows:
    - Added PRIMARY KEY constraint on id field
    - Removed column species
    - Added column species_id 
    - Added foreign key constraint referencing species table
    - Added column owner_id 
    - Added a foreign key constraint referencing the owners table
 - Inserted owner records into the owners table
 - Inserted species records into the species table
 - Updated animals records so they have species_id value
 - Updated animals records so they have owner information (owner_id)
 - Wrote queries (using JOIN) to answer the questions in the activity

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -24,3 +24,33 @@ INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg, s
...
+INSERT INTO owners(full_name, age)
+    VALUES ('Sam Smith', 34);
+INSERT INTO owners(full_name, age)
+    VALUES ('Jennifer Orwell', 19);
+INSERT INTO owners(full_name, age)
+    VALUES ('Bob', 45);
+INSERT INTO owners(full_name, age)
+    VALUES ('Melody Pond', 77);
+INSERT INTO owners(full_name, age)
+    VALUES ('Dean Wincheste', 14);
```
Comment (mmsesay) [inline data.sql]: - [ ] I believe you made a typo here by leaving the letter `r` out of the owner's last name. The owner's name should be `Dean Winchester`

File: data.sql [hunk 2]
```diff
@@ -24,3 +24,33 @@ INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg, s
...
+INSERT INTO owners(full_name, age)
+    VALUES ('Jodie Whittake', 38);
```
Comment (mmsesay) [inline data.sql]: - [ ] I believe you made a typo here by leaving the letter `r` out of the owner's last name. The owner's name should be `Jodie Whittaker`

File: data.sql [hunk 3]
```diff
@@ -24,3 +24,33 @@ INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg, s
...
+INSERT INTO species(name)
+    VALUES ('Digimon');
+
+-- Modify your inserted animals so it includes the species_id value: 
+UPDATE animals SET species_id=2 where name LIKE '%mon';
+UPDATE animals SET species_id=1 where name NOT LIKE '%mon';
+
+-- Modify your inserted animals to include owner information (owner_id): 
+UPDATE animals SET owners_id=(SELECT id from owners where name like 'Jodie Whittake')
+WHERE name='Agumon';
```
Comment (mmsesay) [inline data.sql]: - [ ] The owner for animal `Agumon` should be `Sam Smith` instead of `Jodie Whittaker`

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @gtekle, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gtekle** [comment]
Hi @mmsesay 👋 
Thank you very much for the detailed feedback. I will address all of them shortly.
🙇 thanks again 🙇

**Review by t-yanick — APPROVED** [review]
Hi @gtekle, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gtekle** [comment]
Hi @t-yanick 
Thank you very much. :bow:

**Pull request merged**